src/utils/ImportExportHelper.ts

```diff
@@ -0,0 +1,59 @@
+// Utility methods to call import/export Jahia API
+
+// http://localhost:8080/cms/export/default/export.zip?exportformat=site&live=true&users=true&path=/groups&path=/settings/mail-server-1&sitebox=digitall&exportPath=abc
+// Export
+
+import RequestOptions = Cypress.RequestOptions;
+import {JahiaServer} from '../support';
+
+const API_NAME = '/cms/export';
+
+export type ExportParameters = {
+    viewContent?: boolean;
+    viewVersion?: boolean;
+    viewAcl?: boolean;
+    viewLinks?: boolean;
+    viewMetadata?: boolean;
+    viewWorkflow?: boolean;
+    exportPath?: string;
+    exportformat: 'all' | 'site' | 'xml' | 'zip';
+    root?: string;
+    live?: boolean;
+    users?: boolean;
+    sitebox?: string[];
+    paths?: string[];
+    cleanup?: 'template' | 'simple';
+    filesToZip?: string;
+}
+
+const request = (workspace: string, nodePath: string, exportFormat: string, params: string, jahiaServer: JahiaServer): Partial<RequestOptions> => ({
```

Comment:
shouldn't params be either an object or an array and be converted to a string at the very last moment (i.e. the responsibility of the request method to transform the data it received into the right format)?

At the same time, I'm not sure that it's really needed to have a separate request method since it's not used anywhere else.

Reply:
it's now an object